templates/includes/sidebar.html

```diff
@@ -0,0 +1,8 @@
+<div class="sidebar">
+  <ul class="sidebar-nav">
+    <li><a href="{% url 'taxi:index' %}">Home page</a></li>
+    <li><a href="">Manufacturers</a></li>
+    <li><a href="">Cars</a></li>

```

Comment:
The 'Cars' link is missing a URL. Consider using the Django template tag {% url 'app_name:view_name' %} to generate the correct URL.